### DISCUSSION TITLE
#13. VATZ biweekly meeting at `2022-07-20 08:00:00 CT`

### DISCUSSION THREAD

**Discussion by xellos00** [dsrvlabs/vatz#172] [General]
![image](https://user-images.githubusercontent.com/6308023/178072681-048e4ca7-8338-49e2-9ea9-6a06ab77de6d.png)

In Previous
> Overall 4/16 issues. 

- Finished
  - #142
  - #164
  - #150
  - #158
  
- Move to Sprint 13
   - #69 
   - #85 
   - #87 
   - #119 
   - #120 
   - #121 
   - #122 
   - #127 
   - #146 
   - #151 
   - #157 
   - #163 

---

Discussion:
- https://github.com/dsrvlabs/vatz/issues/187#issuecomment-1189699071
- https://github.com/dsrvlabs/vatz/issues/174
- https://github.com/dsrvlabs/vatz/issues/88
- https://github.com/dsrvlabs/vatz/issues/151#issuecomment-1181243187

**Comment by xellos00** [comment]
# 동요

- #151 
- #186 
- #174
- #180

**Comment by rootwarp** [comment]
# 준교
### Issue
- #125
  - #181 
  - #179

### PR
- #182
- #185

**Comment by gnongs** [comment]
## 진홍
- https://github.com/dsrvlabs/vatz/issues/127
- https://github.com/dsrvlabs/vatz/issues/152
- https://github.com/dsrvlabs/vatz/issues/177
- https://github.com/dsrvlabs/vatz/issues/191

**Comment by heejin-github** [comment]
# 희진
- #194
- https://github.com/dsrvlabs/vatz-plugin-solana/pull/17
- https://github.com/dsrvlabs/vatz-plugin-solana/pull/14
- https://github.com/dsrvlabs/vatz-plugin-solana/pull/16
- https://github.com/dsrvlabs/vatz-plugin-solana/pull/18
- https://github.com/dsrvlabs/vatz-plugin-solana/issues/19

**Comment by kim201212** [comment]
# 형기

- #166 
   - https://github.com/dsrvlabs/vatz/pull/190

**Comment by meetrick** [comment]
# 황제
- https://github.com/dsrvlabs/vatz/issues/121
  - https://github.com/dsrvlabs/vatz-plugin-osmosis
- https://github.com/dsrvlabs/vatz/issues/152
  - https://github.com/dsrvlabs/vatz-plugin-relayer
- https://github.com/dsrvlabs/vatz/issues/183
  - https://github.com/dsrvlabs/vatz-plugin-aptos
- https://github.com/dsrvlabs/vatz/issues/184
  - https://github.com/dsrvlabs/vatz-plugin-sei
